### DISCUSSION TITLE
GLTF scenes to Prefabs

### DISCUSSION THREAD

**Discussion by IvLaptev** [atteneder/glTFast#255] [Q&A] [answered]
Hi. I'm trying to pack gltf scenes into assetbundles programmatically. Unfotunatelly, I ran into a problem: I can't find a way to create correct prefabs from gltf models. I have very simple pipeline:

1. Import gltf model
2. Instantiate it
3. Save game object as a prefab

But there is a problem. All prefabs are creating without materials, meshes, animations ect. Is there any right way to create prefab with scripts that will save all attached objects?

**Comment by atteneder** [comment, marked as answer]
Hi @IvLaptev ,

Do you import/instantiate in play mode?

If so, an alternative would be to just copy your glTFs into the Assets folder, which performs a design-time import and creates prefabs.

Regards
